Raygun4UWP/Properties/AssemblyInfo.cs

```diff
@@ -26,7 +26,7 @@
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.1.1.0")]
 [assembly: AssemblyFileVersion("1.1.1.0")]
-[assembly: AssemblyInformationalVersion("1.1.1-rc.1")]
+//[assembly: AssemblyInformationalVersion("1.1.1-rc.1")]

```

Comment:
Was this on purpose?

Reply:
Yes, so that people know in the future how to mark a prerelease